Userland/Applications/NetworkSettings/CertStoreDialog.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2023, Fabian Dellwing <fabian@dellwing.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Applications/NetworkSettings/CertStoreDialog.h>
+#include <Applications/NetworkSettings/CertStoreDialogGML.h>
+#include <LibCrypto/ASN1/PEM.h>
+
+namespace NetworkSettings {
+
+NonnullRefPtr<CertStoreModel> CertStoreModel::create() { return adopt_ref(*new CertStoreModel); }
+
+void CertStoreModel::load()
+{
+    // FIXME: In the future we will allow users to import own CAs, we than need to change this logic
```

Comment:
I feel like this could be worded a bit better:
``
// FIXME: In the future, we will allow users to import their own certificates. To allow this, we would need to change this logic
``